schemas/disease_target_evidence.json

```diff
@@ -1488,6 +1482,52 @@
       "minItems": 1,
       "uniqueItems": true
     },
+    "assays": {
+      "type": "array",
+      "description": "Assays used in the study",
+      "items": {
+        "type": "object",
+        "description": "Description and result of one assay",
+        "properties": {
+          "shortName": {
+            "type": "string",
+            "description": "Short name of the assay.",
+            "examples": [
+              "CellTiter-Glo"
+            ]
+          },
+          "description": {
+            "type": "string",
+            "description": "Description of the assay",
+            "examples": [
+              "Homogeneous method of quantifying the presence of ATP, an indicator of metabolically active cells, and using this as a proxy to determine the number of viable cells."
+            ]
+          },
+          "isHit": {
+            "type": "boolean",
+            "description": "Indicating if the assay was positive or negative for the target",
+            "enum": [
+              true,
+              false
+            ]
+          }
+        },
+        "required": [
+          "shortName",
+          "description",
+          "isHit"
+        ],
+        "additionalProperties": false
+      },
+      "uniqueItems": true
+    },
+    "assessment": {
```

Comment:
Would it make sense for this to be an enum?

Reply:
Unfortunately not. The VL assessment is not assessment in a very direct sense (eg. `target validated` or `target not validated`. This field will contain an "interpreted summary" of the validation assays, so would be change from project to project (there are different assays in the colo and breast project).